src/App.js

```diff
@@ -14,11 +14,51 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  render() {
+    const { selectedGood } = this.state;
+
+    return (
+      <>
+        {selectedGood ? (
+          <h1>
+            {`Selected good: - ${selectedGood} is selected`}
+            <button
+              type="button"
+              onClick={() => {
+                this.setState({ selectedGood: null });
+              }}
+            >
+              x
+            </button>
+          </h1>
+        ) : (
+          <h1>No value selected</h1>
+        )}
+
+        {goodsFromServer.map(n => (
```

Comment:
```suggestion
        {goodsFromServer.map(good => (
```
Let's give more meaningful names